### PR TITLE
Java: Add Zcount command. (Sorted Set Group)

### PR DESCRIPTION
**Pull request by SanHalacogluImproving**

### CODE REVIEW

File: java/client/src/main/java/glide/api/models/commands/RangeOptions.java [hunk 1]
```diff
@@ -10,9 +10,10 @@
 import lombok.RequiredArgsConstructor;
 
 /**
- * Arguments for {@link SortedSetBaseCommands#zrange} and {@link
- * SortedSetBaseCommands#zrangeWithScores}
+ * Arguments for {@link SortedSetBaseCommands#zcount}, {@link SortedSetBaseCommands#zrange} and
```
Comment (acarbonetto) [inline java/client/src/main/java/glide/api/models/commands/RangeOptions.java:13]:
```suggestion
 * Arguments for {@link SortedSetBaseCommands#zcount}, {@link SortedSetBaseCommands#zrange}, and
```
https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/

File: java/integTest/src/test/java/glide/SharedCommandTests.java [hunk 2]
```diff
@@ -1364,4 +1364,55 @@ public void zrange_with_different_types_of_keys(BaseClient client) {
...
+    public void zcount(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+        Map<String, Double> membersScores = Map.of("one", 1.0, "two", 2.0, "three", 3.0);
+        assertEquals(3, client.zadd(key1, membersScores).get());
+
+        assertEquals(
+                3,
+                client
+                        .zcount(key1, InfScoreBound.NEGATIVE_INFINITY, InfScoreBound.POSITIVE_INFINITY)
```
Comment (acarbonetto) [inline java/integTest/src/test/java/glide/SharedCommandTests.java:1380]: If you import the static fields... 
```suggestion
                        .zcount(key1, NEGATIVE_INFINITY, POSITIVE_INFINITY)
```

Comment (acarbonetto) [reply java/integTest/src/test/java/glide/SharedCommandTests.java:1380]: This might clean up your line...

**Review by acarbonetto — APPROVED** [review]

**Review by jonathanl-bq — APPROVED** [review]

File: java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java [hunk 3]
```diff
@@ -456,4 +459,31 @@ CompletableFuture<Map<String, Double>> zrangeWithScores(
...
+     * @param key The key of the sorted set.
+     * @param minScore The minimum score to count from. Can be an implementation of {@link
+     *     InfScoreBound} representing positive/negative infinity, or {@link ScoreBoundary}
+     *     representing a specific score and inclusivity.
+     * @param maxScore The maximum score to count up to. Can be an implementation of {@link
+     *     InfScoreBound} representing positive/negative infinity, or {@link ScoreBoundary}
+     *     representing a specific score and inclusivity.
+     * @return The number of members in the specified score range.<br>
+     *     If <code>key</code> does not exist, it is treated as an empty sorted set, and the command
+     *     returns 0.<br>
```
Comment (Yury-Fridlyand) [inline java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java:477]: wrap 0 with `code`

File: java/client/src/main/java/glide/api/models/BaseTransaction.java [hunk 4]
```diff
@@ -1410,6 +1414,29 @@ public T zrankWithScore(@NonNull String key, @NonNull String member) {
...
+     * @param key The key of the sorted set.
+     * @param minScore The minimum score to count from. Can be an implementation of {@link
+     *     InfScoreBound} representing positive/negative infinity, or {@link ScoreBoundary}
+     *     representing a specific score and inclusivity.
+     * @param maxScore The maximum score to count up to. Can be an implementation of {@link
+     *     InfScoreBound} representing positive/negative infinity, or {@link ScoreBoundary}
+     *     representing a specific score and inclusivity.
+     * @return Command Response - The number of members in the specified score range.<br>
+     *     If <code>key</code> does not exist, it is treated as an empty sorted set, and the command
+     *     returns 0.<br>
```
Comment (Yury-Fridlyand) [inline java/client/src/main/java/glide/api/models/BaseTransaction.java:1431]: same

**Review by Yury-Fridlyand — APPROVED** [review]

File: java/integTest/src/test/java/glide/SharedCommandTests.java (hunk 2 above)
Comment (SanHalacogluImproving) [reply java/integTest/src/test/java/glide/SharedCommandTests.java:1380]: I think this should not be imported as we also have InfLexBound

**Review by Yury-Fridlyand — APPROVED** [review]

**Pull request merged**